libafl/src/executors/forkserver.rs

```diff
@@ -733,6 +742,9 @@ impl<'a, SP> ForkserverExecutorBuilder<'a, SP> {
             if item.as_ref() == "@@" && use_stdin {
                 use_stdin = false;
                 res.push(OsString::from(".cur_input"));
+                if self.uses_shmem_testcase {
+                    println!("WARNING: @@ is used but uses_shmem_testcase is set, check your configuration");
```

Comment:
Because it's probably unwanted. If I, a user, specify @@, I expect fuzz input to be passed via a file. If I want to use shmem, I have to configure this inside my harness and make use of the ENV variable, making @@ unnecessary.

Reply:
Not really, it's the only way to use unicornafl, for example